source/widgets/visualize/distributions.md

```diff
@@ -13,29 +13,42 @@ Displays value distributions for a single attribute.
 - Data: data with an additional column showing whether an instance is selected
 - Histogram Data: bins and instance counts from the histogram
 
-The **Distributions** widget displays the [value distribution](https://en.wikipedia.org/wiki/Frequency_distribution) of discrete or continuous attributes. If the data contains a class variable, distributions may be conditioned on the class.
+The **Distributions** widget displays the [value distribution](https://en.wikipedia.org/wiki/Frequency_distribution) of discrete or continuous attributes. If the data contains a class variable, distributions may be split by the class.
```

Comment:
Shall we call them discrete and continuous or categoric and numeric? The former came from the ML tradition and is everywhere in code, while in UI we are, I guess, preferring the latter?